batchflow/models/torch/layers/hamburger.py

```diff
@@ -0,0 +1,127 @@
+""" Hamburger layer.
+Zhengyang Geng et al. "`Is attention better than matrix decomposition? <https://arxiv.org/abs/2109.04553>`_"
+"""
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+
+
+class Hamburger(nn.Module):
+    """ Hamburger layer: attention by matrix decomposition.
+    The main idea is to use classic matrix decomposition algorithms inside the layer during both train and inference.
```

Comment:
- it's not particularly 'classis' MD, more like its differentiable version 
- algorithms => alrorithm?